src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -548,8 +545,8 @@ class NetworkOPsImp final
 
     std::atomic<OperatingMode> mMode;
 
-    std::atomic <bool> mNeedNetworkLedger;
-    bool m_amendmentBlocked;
+    std::atomic <bool> needNetworkLedger_ {false};
+    std::atomic <bool> amendmentBlocked_ {false};
```

Comment:
My $0.02: Reading an `atomic_flag` sets it to true, so I'm not sure how'd we'd implement `bool isNeedNetworkLedger ()` with `atomic_flag`. I'd say we want `atomic<bool>` here and not `atomic_flag`